phaser-react/src/states/flow1/Catalyst.jsx

```diff
@@ -6,21 +6,23 @@ export default function Catalyst({ state, send }) {
   if (state.matches('catalyst_earth')) {
     return (
       <div>
-        Catalyst Earth
-        <button onClick={() => send({type: "CONTINUE"})}>continue</button>
+        <h1> Catalyst Earth </h1>
+        <p> Political corruption and rumors of fraud in the most recent election has cause much social unrest in Costa Rica. There are protests daily, and they often end in violence. Rivaling cartels are using the circumstances to gain political power and control over more territory. </p>
+        <button onClick={() => send({type: "CONTINUE"})}> Continue</button>
       </div>
     );
   }
   if (state.matches('catalyst_country')) {
     return (
       <div>
-        Catalyst Country
-        <button onClick={() => send({type: "CONTINUE"})}>continue</button>
+        <h1> Catalyst Country </h1>
+        <p> Having just graduated high school, you are envisioning a better future as the current landscape becomes more hopeless everyday. You dream of escaping to somewhere with better opportunities. </p>
+        <button onClick={() => send({type: "CONTINUE"})}> Continue</button>
       </div>
     );
   }
   
   return (
-    <div>catalyst</div>
+    <div> Error: create error page later </div>
   )
```

Comment:
Just a suggestion, but instead of having 2 if statements and a return housing the error message, you could have 1 if statement chained with an else if and else statement.
```
if (state.matches('catalyst_earth')) {
     return code
} else if (state.matches('catalyst_country') {
     return code
} else {
     return error message
}
```

Reply:
great catch! I will fix that asap.